### ISSUE TITLE
Couldn't find Spree::Page

### ISSUE THREAD

**Issue by octoxan** [solidusio-contrib/solidus_static_content#33]
Installed, ran migrations, created a page in the back end, but when trying to visit the slug I get...

```
ActiveRecord::RecordNotFound in Spree::StaticContentController#show

Couldn't find Spree::Page

def find_by!(*args) # :nodoc:
    find_by(*args) || raise(RecordNotFound.new("Couldn't find #{name}", name))
end

def initialize_generated_modules # :nodoc:
```

Edit: Just noticed it seems to be unchecking my store check box on every save.

**Comment by octoxan** [comment]
Doesn't ever save anything to the spree_pages_stores table.

**Comment by octoxan** [comment]
Manually adding a row to that table makes the page visible.

**Comment by gildardoperez** [comment]
@octoxan 

run:
`> bundle exec rake railties:install:migrations FROM=spree_static_content`

**Comment by gildardoperez** [comment]
@octoxan can you please share your solution (command) used.

**Comment by octoxan** [comment]
@gildardoperez I just used a database editor to manually add a row to the spree_pages_stores with the correct page and store ID, since this gem is broken currently and never updates that table. I'm unsure of a real fix at the moment.

**Comment by jacobherrington** [comment]
@octoxan I am not familiar with this repo, but I will try to take a look at your PR. Thanks!

**Comment by xbold** [comment]
Confirmed this is broken out of the box but that work fixes it for me.

**Comment by octoxan** [comment]
@jacobherrington thanks! Appreciate it!

**Comment by octoxan** [comment]
Fixed in #34 which is now merged.
